Patch out newgem since we don't need it to install and it currently
isn't available in Gentoo.

--- Rakefile.orig	2010-02-25 07:56:59.000000000 +0100
+++ Rakefile	2010-02-25 07:56:28.000000000 +0100
@@ -1,4 +1,4 @@
-%w[rubygems rake rake/clean fileutils newgem rubigen hoe].each { |f| require f }
+%w[rubygems rake rake/clean fileutils rubigen hoe].each { |f| require f }
 $LOAD_PATH << File.dirname(__FILE__) + '/lib'
 require 'oauth'
 require 'oauth/version'
@@ -18,7 +18,6 @@
     ['ruby-hmac','>= 0.3.1']
   ]
   p.extra_dev_deps = [
-    ['newgem', ">= #{::Newgem::VERSION}"],
     ['actionpack'],
     ['rack']
   ]
@@ -29,7 +28,6 @@
   p.rsync_args = '-av --delete --ignore-errors'
 end
 
-require 'newgem/tasks' # load /tasks/*.rake
 Dir['tasks/**/*.rake'].each { |t| load t }
 
 # TODO - want other tests/tasks run by default? Add them to the list
